cmd/earthly/main.go

```diff
@@ -72,6 +72,13 @@ import (
 	"github.com/earthly/earthly/variables"
 )
 
+const (
+	// DefaultContainerName is the name of the buildkitd container.
+	DefaultContainerName = "earthly-buildkitd"
+	// DefaultVolumeName is the name of the docker volume used for storing the cache.
+	DefaultVolumeName = "earthly-cache"
```

Comment:
Might want to make these variable names a little more specific given that they're no longer under the buildkitd package: `DefaultBuildkitContainerName` & `DefaultBuildkitVolumeName`?